### DISCUSSION TITLE
How to leave a black gap between pages when converting a PDF to an image

### DISCUSSION THREAD

**Discussion by adscott1982** [dlemstra/Magick.NET#703] [Help] [answered]
I started using the library today, and it is working pretty well. I am converting a PDF to an image and displaying it in WPF. 

One thing I want to do is display a gap between pages with a black bar.

I had tried to do this by resizing each image in the collection of images returned from reading in the PDF. However it either didn't work or the default background is not black. 

I also looked at trying to create a blank black image and inserting it in the collection, but I don't see a way of creating a new uninitialised black MagickImage.

I imagine this is actually quite an easy operation and I don't know the API well enough yet.

For someone more experienced, what is the simplest way to achieve what I want?

**Comment by dlemstra** [comment, marked as answer]
You can create an "empty" black image like this; `var image = new MagickImage(MagickColors.Black, width, height)`

**Reply by adscott1982** [reply]
Perfect! Thank you

![image](https://user-images.githubusercontent.com/12865303/88078604-8ec2d700-cb74-11ea-8f2f-5add05c440ce.png)
